extension/Makefile

```
@@ -39,7 +39,7 @@ setup.dependencies: install-pg_cron install-pgvector install-pgmq install-vector
 setup.shared_preload_libraries:
 	echo "shared_preload_libraries = 'pg_cron, vectorize'" >> ~/.pgrx/data-${PG_VERSION}/postgresql.conf
 setup.urls:
-	echo "vectorize.embedding_service_url = 'http://localhost:3000/v1'" >> ~/.pgrx/data-${PG_VERSION}/postgresql.conf
+	echo "vectorize.embedding_service_url = 'http://localhost:3000/v1/embeddings'" >> ~/.pgrx/data-${PG_VERSION}/postgresql.conf
```

This document is already correct for the latest version of pg_vectorize. https://github.com/tembo-io/pg_vectorize/blob/aac399976470e96543a49e4efd9d9d27b38761f2/extension/Makefile#L41-L43

```suggestion
	echo "vectorize.embedding_service_url = 'http://localhost:3000/v1'" >> ~/.pgrx/data-${PG_VERSION}/postgresql.conf
```